### DISCUSSION TITLE
Not able to verify contract

### DISCUSSION THREAD

**Discussion by Raritica** [hashlips-lab/nft-erc721-collection#268] [Q&A]
Hi, thank you for your great job.
Could you help me to figur out this problem?

I followed the video step by step and deploy the contract succesfully but when I tried to verify it, no matter whitch way I chose, directly deployment or truffle deployment, there comes an error as following.

```
yarn verify 0x8CDB9703546d7DD222b8BCcB2f0a75f2648B0B16 --network truffle
yarn run v1.22.15
$ hardhat verify --constructor-args config/ContractArguments.ts 0x8CDB9703546d7DD222b8BCcB2f0a75f2648B0B16 --network truffle
Nothing to compile
No need to generate any newer typings.
Error in plugin @nomiclabs/hardhat-etherscan: Failed to obtain list of solc versions. Reason: Connect Timeout Error

For more info run Hardhat with --show-stack-traces
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Pls help, thank you.

**Comment by yzh617** [comment]
Same questions, I've tried many times, Pls help, Thank you

Error in plugin @nomiclabs/hardhat-etherscan: Failed to send contract verification request.
Endpoint URL: https://api-rinkeby.etherscan.io/api
Reason: read ECONNRESET

For more info run Hardhat with --show-stack-traces
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by andestajohan** [comment]
Same here, i tried 10 times already and it saying the API key in INVALID, we need some help here, i had no problem when deploying on Mumbai net, the moment i deployed on Polygon mainnet, this error keeps coming out. HELP please
#<img width="709" alt="image" src="https://user-images.githubusercontent.com/106788001/173246924-7b7fac43-d0f6-4994-ac6e-4b17aaffa016.png">

**Reply by bolisteward** [reply]
How you solved it ?

**Comment by wee4050** [comment]
Uh... I'm having trouble validating the contract:
Nothing to compile
No need to generate any newer typings.
Error in plugin @nomiclabs/hardhat-etherscan: Failed to send contract verification request.
Endpoint URL: https://api-rinkeby.etherscan.io/api
Reason: Connect Timeout Error
![屏幕截图 2022-06-16 112509](https://user-images.githubusercontent.com/96134975/173984448-ee569fec-bf31-4932-9570-2c4563fd359d.png)
